lib/rules/web/admin_console/4.5/goto.xyaml

```diff
@@ -6,6 +6,7 @@ goto_project_status:
   url: status/ns/<project>
 goto_resource_page_under_default_project:
   url: k8s/ns/default/<resource_url_name>
+  action: wait_box_loaded
```

Comment:
added `wait_box_loaded` to make sure page is loaded completely before going next step otherwise we will see errors in Runner-v3 #92866